### PR TITLE
Add Scorecard result viewer

### PR DESCRIPTION
**Pull request by cynthia-sg**
Related to ossf/scorecard/issues/2979

### CODE REVIEW

**Comment by spencerschrock** [comment]
Thanks for sending this over. I am getting some help reviewing this, so it may take a little more time. I have enabled the deploy preview which can be seen at:

https://64798d93e9804c0008a21b9a--ossf-scorecard.netlify.app/viewer/?platform=github.com&org=ossf&repo=scorecard

File: scorecards-site/static/viewer/index.html [hunk 1]
```diff
@@ -0,0 +1,1318 @@
...
+            </svg>
+          </div>
+        </td>
+        <td class="ps-0">
+          <div class="d-flex flex-row">
+            <div class="d-flex flex-column justify-content-between table-cell">
+              <div class="d-flex flex-row align-items-center main-title">
+                {{getRepoLogo}}
+                <div class="fw-semibold text-truncate">
+                  <a class="btn-link repo-name" href="https://{{data.repo.name}}" target="_blank" rel="noopener noreferrer">
```
[the remark this answers is not in the document]
Comment (tegioz) [reply scorecards-site/static/viewer/index.html:809]: They are escaped, yes, but I think [there is one case](https://github.com/cynthia-sg/scorecard-webapp/blob/f69ec11402089da7b409fbc7b182f3af27454933/scorecards-site/static/viewer/index.html#LL864C108-L864C1089) in which code could be injected if the scorecard results were compromised, as Handlebars does not escape JavaScript strings. @cynthia-sg will push a fix for this 🙂

File: scorecards-site/static/viewer/index.html [hunk 2]
```diff
@@ -0,0 +1,1318 @@
...
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="scorecard-report">
+      <div class="table-wrapper">
+        <table class="table position-relative">
+          <thead>
+            <tr>
+              <th colspan="2" class="title">
+                <div class="d-flex flex-row align-items-center justify-content-center">
+                  <div class="logo">
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 590 1000">
```
[the remark this answers is not in the document]
Comment (tegioz) [reply scorecards-site/static/viewer/index.html:702]: Just for the convenience of having a single file. But it'd probably be easier to maintain if it was in a separate file. That would also make it easier to ask the browser/CDN to cache the SVG for longer if you'd like.

File: scorecards-site/static/viewer/index.html [hunk 3]
```diff
@@ -0,0 +1,1318 @@
...
+        if (x === undefined) return 0;
+        return x * y <= 0 ? 1 : x * y;
+      });
+
+      Handlebars.registerHelper('scoreFormat', (x) => {
+        if (x === undefined) return '-';
+        return x.toFixed(1);
+      });
+
+      Handlebars.registerHelper('replaceDashes', (x) => {
```
[the remark this answers is not in the document]
Comment (spencerschrock) [reply scorecards-site/static/viewer/index.html:1103]: Yeah, let's remove this for consistency purposes.

**Review by spencerschrock — APPROVED** [review]
Thanks again to both of you!

**Pull request merged**